base/tomcat-9.0/conf/server.xml

```diff
@@ -126,7 +126,6 @@
                ocspMinCacheEntryDuration="7200"
                ocspMaxCacheEntryDuration="14400"
                ocspTimeout="10"
-               serverCertNickFile="[PKI_INSTANCE_PATH]/conf/serverCertNick.conf"
```

Comment:
Since the file is generated, we could remove it from `base/server/share/conf/serverCertNick.conf`

Reply:
Good catch! Removed.